### ISSUE TITLE
Adding an IPv6 only seed fails to connect

### ISSUE THREAD

**Issue by Nemo157** [radicle-dev/radicle-link#472]
I setup a seed node on my server with only a public IPv6 address specified (`hybjnt4saqzz77zih7b6jp1ro8kzq3w6owh6nuawrkw6y6js4ssrd4@radicle.nemo157.com:4001` if you want to test connecting, I'll leave it setup on IPv6 only for now), when I add this seed to my local `radicle-proxy` (built from tag `0.1.4`) it fails to connect:

```
 INFO  librad::net::protocol > establishing connection remote.id=hybjnt4saqzz77zih7b6jp1ro8kzq3w6owh6nuawrkw6y6js4ssrd4 remote.addr=[2a02:c207:2020:2848:a08f:2e80:b608:87b6]:4001
 ERROR quinn::builders       > I/O error: Address family not supported by protocol (os error 97)
 WARN  librad::net::protocol > could not connect to hybjnt4saqzz77zih7b6jp1ro8kzq3w6owh6nuawrkw6y6js4ssrd4 at [2a02:c207:2020:2848:a08f:2e80:b608:87b6]:4001: closed
```

**Comment by Nemo157** [comment]
Updating my local `radicle-proxy` to bind to `[::]:0` here allowed it to connect:

https://github.com/radicle-dev/radicle-upstream/blob/c111d6eaa7963bac9d30ad0708115103d8267d61/proxy/coco/src/config.rs#L22

It seems like this needs support for binding to multiple sockets.

**Comment by Nemo157** [comment]
Actually, so that I can continue testing stuff I've moved my seed onto IPv4 now, if there are fixes to be tested I can switch it back as needed.

**Comment by kim** [comment]
> It seems like this needs support for binding to multiple sockets.

Yes, on my radar. Dual-stack sockets are, however, not portable, so this would -- as you say -- involve running more than one QUIC stack. Which complicates things quite considerably. I'd say let's get IPv4 stable, and then see.
